<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScreeningManagerClass</class>
 <widget class="QWidget" name="ScreeningManagerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>780</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>780</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ScreeningManager</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background-color: rgb(253, 255, 220)
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_5">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0" columnstretch="0,0">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <property name="horizontalSpacing">
      <number>9</number>
     </property>
     <property name="verticalSpacing">
      <number>0</number>
     </property>
     <item row="2" column="1" alignment="Qt::AlignRight|Qt::AlignBottom">
      <widget class="QPushButton" name="pushButton_return">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>35</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: rgb(255, 255, 242);
    border: 1px solid black;
    border-radius: 6px;
    padding: 6px;
}

QPushButton:hover {
    background-color: rgb(182, 178, 135);
}

QPushButton:pressed {
    background-color: #d0d0d0;
}</string>
       </property>
       <property name="text">
        <string>Return</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="minimumSize">
        <size>
         <width>385</width>
         <height>523</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>385</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	border: 3px solid black;
	image: url(:/ProjectionScreen/Resource files/posters/cinemaposter.png);
	
}</string>
       </property>
       <property name="lineWidth">
        <number>0</number>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QGroupBox" name="groupBox_3">
       <property name="styleSheet">
        <string notr="true">QGroupBox {
    border: 2px solid black;
    border-radius: 5px;
    margin-top: 20px;
    padding: 10px;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top center; /* pozycja tytułu */
    padding: 0 5px;
    background-color: transparent;
    color: black;
}</string>
       </property>
       <property name="title">
        <string>Screening - delete</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Screening id:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_screening_id">
          <property name="minimumSize">
           <size>
            <width>119</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>90</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QComboBox {
   	background-color: rgb(255, 255, 242);
    border: 1px solid black;
    border-radius: 3px;
    padding: 1px 18px 1px 3px;
    min-width: 6em;
}

QComboBox:hover {
    border: 1px solid #aaa;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: right;
    width: 20px;
    border-left-width: 1px;
    border-left-color: black;
    border-left-style: solid;
    border-top-right-radius: 3px;
    border-bottom-right-radius: 3px;
  background-color: rgb(194, 200, 140);
}

QComboBox::down-arrow {
    image: url(:/ProjectionScreen/Resource files/icons/down2.png);
	width: 16px;
	height: 16px;
}

QComboBox::down-arrow:on {
    top: 1px;
    left: 1px;
}</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_delete_screening">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>35</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
       background-color: rgb(182, 178, 135);
    border: 1px solid black;
    border-radius: 6px;
    padding: 6px;
}

QPushButton:hover {

 background-color: rgb(255, 255, 242);
}

QPushButton:pressed {
    background-color: #d0d0d0;
}</string>
          </property>
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="maximumSize">
        <size>
         <width>395</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QGroupBox {
    border: 2px solid black;
    border-radius: 5px;
    margin-top: 20px;
    padding: 10px;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top center; /* pozycja tytułu */
    padding: 0 5px;
    background-color: transparent;
    color: black;
}</string>
       </property>
       <property name="title">
        <string>Screening addition - Movie/ hall_id</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_movie">
            <property name="font">
             <font>
              <family>Arial</family>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Movie:</string>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignLeft">
           <widget class="QComboBox" name="comboBox_movies">
            <property name="minimumSize">
             <size>
              <width>270</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>260</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QComboBox {
   	background-color: rgb(255, 255, 242);
    border: 1px solid black;
    border-radius: 3px;
    padding: 1px 18px 1px 3px;
    min-width: 280px
}

QComboBox:hover {
    border: 1px solid #aaa;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: right;
    width: 20px;
    border-left-width: 1px;
    border-left-color: black;
    border-left-style: solid;
    border-top-right-radius: 3px;
    border-bottom-right-radius: 3px;
    background-color: rgb(194, 200, 140);
}

QComboBox::down-arrow {
    image: url(:/ProjectionScreen/Resource files/icons/down2.png);
	width: 16px;
	height: 16px;
}

QComboBox::down-arrow:on {
    top: 1px;
    left: 1px;
}</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Arial</family>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="text">
             <string>hall_id:</string>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignLeft">
           <widget class="QComboBox" name="comboBox_hall">
            <property name="minimumSize">
             <size>
              <width>270</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>270</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QComboBox {
   	background-color: rgb(255, 255, 242);
    border: 1px solid black;
    border-radius: 3px;
    padding: 1px 18px 1px 3px;
    min-width: 280px;
}

QComboBox:hover {
    border: 1px solid #aaa;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: right;
    width: 20px;
    border-left-width: 1px;
    border-left-color: black;
    border-left-style: solid;
    border-top-right-radius: 3px;
    border-bottom-right-radius: 3px;
background-color: rgb(194, 200, 140);
}

QComboBox::down-arrow {
    image: url(:/ProjectionScreen/Resource files/icons/down2.png);
	width: 16px;
	height: 16px;
}

QComboBox::down-arrow:on {
    top: 1px;
    left: 1px;
}
QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: right;
    width: 20px;
    border-left-width: 1px;
    border-left-color: #ccc;
    border-left-style: solid;
    border-top-right-radius: 3px;
    border-bottom-right-radius: 3px;
   background-color: rgb(194, 200, 140);
}

QComboBox::down-arrow {
    image: url(:/ProjectionScreen/Resource files/icons/down2.png);
	width: 16px;
	height: 16px;
}
QComboBox::down-arrow:on {
    top: 1px;
    left: 1px;
}</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QGroupBox" name="groupBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>150</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QGroupBox {
    border: 2px solid black;
    border-radius: 5px;
    margin-top: 20px;
    padding: 10px;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top center; 
    padding: 0 5px;
    background-color: transparent;
    color: black;
}</string>
       </property>
       <property name="title">
        <string>Screening addition - date/time</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QFrame" name="mainFrame">
            <property name="minimumSize">
             <size>
              <width>156</width>
              <height>45</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>156</width>
              <height>45</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">#mainFrame {
    border: 2px solid black;
    border-radius: 10px;
    padding: 5px;
    background-color:rgb(255, 255, 242);
}</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <property name="spacing">
              <number>2</number>
             </property>
             <property name="leftMargin">
              <number>2</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>2</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QDateEdit" name="dateEdit">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>30</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>-1</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">QDateEdit::up-button { width: 0px; height: 0px; border: none;
}
QDateEdit::down-button { width: 0px; height: 0px; border: none;}

#dateEdit {
    font-size: 17px;
    padding: 5px;
    border: 1px solid black;
    border-radius: 5px;
	background-color: rgb(255, 255, 242);
}</string>
               </property>
               <property name="wrapping">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_calendar">
               <property name="minimumSize">
                <size>
                 <width>30</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>30</width>
                 <height>30</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton {
        border: none;        
        background: transparent;
	    image: url(:/ProjectionScreen/Resource files/icons/calendar.png);
}

   </string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_12_00">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>35</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
    background-color: rgb(255, 255, 242);
    border: 1px solid black;
    border-radius: 6px;
    padding: 6px;
}

QPushButton:hover {
    background-color: rgb(182, 178, 135);
}

QPushButton:pressed {
    background-color: #d0d0d0;
}</string>
            </property>
            <property name="text">
             <string>12:00</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_15_00">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>35</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
    background-color: rgb(255, 255, 242);
    border: 1px solid black;
    border-radius: 6px;
    padding: 6px;
}

QPushButton:hover {
    background-color: rgb(182, 178, 135);
}

QPushButton:pressed {
    background-color: #d0d0d0;
}</string>
            </property>
            <property name="text">
             <string>15:00</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_18_00">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>35</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
    background-color: rgb(255, 255, 242);
    border: 1px solid black;
    border-radius: 6px;
    padding: 6px;
}

QPushButton:hover {
    background-color: rgb(182, 178, 135);
}

QPushButton:pressed {
    background-color: #d0d0d0;
}</string>
            </property>
            <property name="text">
             <string>18:00</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_21_00">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>35</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
    background-color: rgb(255, 255, 242);
    border: 1px solid black;
    border-radius: 6px;
    padding: 6px;
}

QPushButton:hover {
    background-color: rgb(182, 178, 135);
}

QPushButton:pressed {
    background-color: #d0d0d0;
}</string>
            </property>
            <property name="text">
             <string>21:00</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item alignment="Qt::AlignHCenter">
         <widget class="QPushButton" name="pushButton_add_screening">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
 
 background-color: rgb(182, 178, 135);
    border: 1px solid black;
    border-radius: 6px;
    padding: 6px;
}

QPushButton:hover {
   background-color: rgb(255, 255, 242);
}

QPushButton:pressed {
    background-color: #d0d0d0;
}</string>
          </property>
          <property name="text">
           <string>Add movie</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QGroupBox" name="groupBox_4">
       <property name="styleSheet">
        <string notr="true">QGroupBox {
    border: 2px solid black;
    border-radius: 5px;
    margin-top: 20px;
    padding: 10px;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top center; 
    padding: 0 5px;
    background-color: transparent;
    color: black;
}</string>
       </property>
       <property name="title">
        <string>Screenings</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QTableView" name="screening_table">
          <property name="minimumSize">
           <size>
            <width>280</width>
            <height>300</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QTableView {
    background-color: rgb(253, 255, 220);
    alternate-background-color: rgb(240, 248, 255);
    gridline-color: rgb(220, 220, 220);
    border: 1px solid black;
}

QHeaderView::section {
    background-color: rgb(182, 178, 135);
    color: black;
    padding: 4px;
    border: 1px solid rgb black;
}

QTableView::item {
    border: none;
    padding: 4px;
}

QTableView::item:selected {
    background-color: rgb(200, 230, 255);
    color: black;
}

QTableView QToolButton {
    background-color: rgb(180, 178, 140); 
    border: none; 
    padding: 4px;
    border-radius: 3px; 
}

QTableView QToolButton:hover {
    background-color: rgb(182, 178, 135);
}</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ProjectionScreen.qrc"/>
 </resources>
 <connections/>
</ui>
